active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
Is interval metadata expected to only live on `Sequence` objects, or are there use-cases for storing interval metadata on other scikit-bio objects (e.g., similar to `metadata` and `positional_metadata` mixins)? I'm asking because that could influence naming. For example, if this were expected to only live on `Sequence` objects, the name `features` may be a more intuitive name for users.